coffee_core/src/coffee.rs

```diff
@@ -281,6 +282,21 @@ impl PluginManager for CoffeeManager {
         self.storage.store(&self.storage_info()).await?;
         Ok(())
     }
+
+    async fn remove_remote(&mut self, name: &str) -> Result<(), CoffeeError> {
+        debug!("remote removing: {}", name);
+        let index = self
+            .repos
+            .iter()
+            .position(|x| &*x.to_owned().name() == name)
+            .unwrap();
+        let repo_path = &self.repos[index].url().path_string;
+        fs::remove_dir_all(repo_path).unwrap();
+        self.repos.remove(index);
+        debug!("remote removed: {}", name);
+        self.storage.store(&self.storage_info()).await?;
+        Ok(())
+    }
```

Comment:
This is not completed, because if you remove the directory as you do we need to remove also the plugin (if any)

After I removed the repository the plugin installed are still there

```
➜  coffee git:(pr80) cargo run -- --network testnet list --remotes 
    Finished dev [unoptimized + debuginfo] target(s) in 0.03s
     Running `target/debug/coffee --network testnet list --remotes`
{
  "plugins": [
    {
      "conf": {
        "plugin": {
          "dependencies": null,
          "deprecated": null,
          "install": "sh -C ./gradlew createRunnableScript\n",
          "lang": "java",
          "main": "btcli4j-gen.sh",
          "name": "btcli4j",
          "version": "0.0.1"
        }
      },
      "lang": "JVM",
      "name": "btcli4j",
      "path": "/home/vincent/.coffee/repositories/lightnind/btcli4j",
      "root_path": "/home/vincent/.coffee/repositories/lightnind/btcli4j"
    },
    {
      "conf": null,
      "lang": "Python",
      "name": "summary",
      "path": "/home/vincent/.coffee/repositories/lightnind/summary",
      "root_path": "/home/vincent/.coffee/repositories/lightnind/summary"
    }
  ],
  "remotes": []
}

```

Reply:
Will the `rm_plugin` function take just the name of the plugin?

I can make an empty function and call it on the plugins installed in the remote repo so that it's automatically used when we implement it.

Reply:
lets wait for the moment :) but guest the rm take the name of the plugin ony